packages/orchestration/src/utils/asset.js

```diff
@@ -0,0 +1,32 @@
+import { denomHash } from './denomHash.js';
+
+/**
+ * @import {ChainHub, CosmosChainInfo, Denom, DenomDetail} from '../types.js';
+ */
+
+/**
+ * Helper function for creating {@link DenomDetail} data for {@link ChainHub}
+ * asset registration.
+ *
+ * @param {Denom} baseDenom
+ * @param {string} baseName
+ * @param {string} [chainName]
+ * @param {Record<string, CosmosChainInfo>} [infoOf]
+ * @param {string} [brandKey]
+ * @returns {[string, DenomDetail & { brandKey?: string }]}
```

Comment:
I'm not comfortable baking `brandKey` into the orch API.

Reply:
Added a TODO that references #10580

Reply:
But that would be a breaking change to a high-visibility API.

Is `brandKey` essential for this PR? Can we just use brands?

Reply:
`brandKey` is an existing convention from `registerChainsAndAssets`, landed on master in [e72782d](https://github.com/Agoric/agoric-sdk/pull/10329/commits/e72782dcc748b9e6a2879179cccf9866718f4e00). The ticket mentions the three contracts expecting it their build scripts.

`assetOn` is not currently exported from `@agoric/orchestration`

Reply:
I'm juggling too much. Somehow I got the impression that `brandKey` hadn't escaped fast-usdc.

`assetOn` is not exported... ok.